### PR TITLE
Added short intro and setup instructions for docusaurus

### PR DESCRIPTION
**Pull request by Mikerniker**
Hello  @jhdalek55 / @Abhijay007
I tried adding a short intro to Uptane and setup instructions in the README.md based on the website and  @Abhijay007 's comments from PR [#84](https://github.com/uptane/uptane.github.io/pull/84). The original Jekyll instructions are still there (under the section I added), since I wasn't sure know if this should be deleted or not. Let me know if you need me to make any changes and if it meets your requirements for a Hacktoberfest PR. Thank you!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,58 @@
...
+   - If the installation went well, the development server will run locally, at ```http://localhost:3000```.
+   - You should see your application running locally with any changes you apply.
+
+
+- If you encounter any issues or have questions, feel free to ask for help.
+
+
+
+
+## Jekyll website
```
Comment (Abhijay007) [inline README.md:55]: I think we don't need this info now as we have migrated the site to the doucusaurus

File: README.md [hunk 2]
```diff
@@ -84,6 +138,8 @@ Add assets, e.g. images or JavaScript, to [`assets`](assets).
 [`favicon.ico`](favicon.ico) should be served from the root of the project.
 Just replace the current one with the `favicon.ico` of your project.
```
Comment (Abhijay007) [inline README.md:140]: till here

Comment (Mikerniker) [reply README.md:140]: Ok, will remove the Jekyll section, should I wait to hear first from the other reviewers for other feedback before making the change or just go ahead and delete it? Thanks!

Comment (Abhijay007) [reply README.md:140]: ya it will be great if we wait for other's review first, so that you can do all the changes in one go

File: README.md [hunk 3]
```diff
@@ -84,6 +138,8 @@ Add assets, e.g. images or JavaScript, to [`assets`](assets).
...
 
+
+
 ## License
```
Comment (hexsecs) [inline README.md:143]: I believe @JustinCappos proposed changing the license. We should probably have a discussion regarding the licensing overall. For now we can keep text as is and fix in a pull request fixing the dedicated issue.

**Review by jhdalek55 — CHANGES_REQUESTED** [review]
I made one suggested change in the intro. Though Uptane is not exclusive to protecting software updates over the air, it was designed for that purpose.

**Comment by Mikerniker** [comment]
Hello @Abhijay007, @hexsecs, @jhdalek55
I've made additional changes to the pull request (i.e. incorporated the change to the Uptane description and deleted the Jekyll section). Could you please review it when you have time? Thank you!

**Review by jhdalek55 — APPROVED** [review]

**Comment by hexsecs** [comment]
Fixes #85

**Pull request merged**